demo/go/README.md

```diff
@@ -2,9 +2,23 @@
 
 This Go module contains demos for processing real-time audio (i.e. microphone) and audio files using the Porcupine wake word engine.
 
-## Installation
-
-MicDemo uses [malgo](https://github.com/gen2brain/malgo) for cross-platform audio capture. It requires `cgo`, which on Windows may mean that you need to install a gcc compiler like [Mingw](http://mingw-w64.org/doku.php) to build it properly. 
+## Requirements
+
+- go 1.16+
+- **Windows**: The demo requires `cgo`, which means that you need to install a gcc compiler like [Mingw](http://mingw-w64.org/doku.php) to build it properly. 
```

Comment:
Is it true that we still need Mingw on Windows now that we've done away with malgo?